### PR TITLE
[#4259] Ignore from toctree *.include.rst files from root folder.

### PR DESCRIPTION
**Pull request by hcs0**
Changes
=======

Added line to exclude `include.rst` files located in an upper folder.

How to try and test the changes
===============================

reviewers: @adiroiban 

Tested the code in server repo and it works.  Please see my additions for readme.rst

### CODE REVIEW

**Comment by hcs0** [comment]
needs-review

File: README.rst [hunk 1]
```diff
@@ -21,13 +21,14 @@ Each change needs a dedicated ticket and a dedicated branch::
 
 Create a PR and request a review.
 
-Run `./paver.sh test_review` to trigger the tests required before merge.
```
Comment (adiroiban) [inline README.rst]: why was this removed?

Comment (hcs0) [inline README.rst]: @adiroiban Do we still need to do this step? Or do you need to do a commit to trigger the tests (or run paver test) ?

Comment (adiroiban) [inline README.rst]: the tests are auto triggered on commit.... but if you need to (re)run the tests without a commit, you can still use paver test_review

**Comment by adiroiban** [comment]
the changes looks good, but we need some "test driven development" here.

Basically, we need at least `paver documenation_standalone` to be available in brink, and add a new root based .include file to make sure that we don't regress on this.

Also, the setup.py file should be updated to reflect the new version.

Thanks for finding the cause of this issue.

I will put this for needs-changes and I will push a commit to add the test and update the version

Thanks again!

File: release-notes.rst [hunk 2]
```diff
@@ -4,6 +4,12 @@ chevah-brink release notes
 Here are the release notes for past brink version.
 
 
+Version 0.63.3, 2017-07-26
+--------------------------
+
+* Exclude include.rst located in an upper folder directory.
```
Comment (adiroiban) [inline release-notes.rst]: I thought that this is for a file located in the root directory.

Now I don't know what you consider to be "an upper" ... but there is only one single top directory and we call it the root :)
